### ISSUE TITLE
Get the a11y scanning process to a sustainable place

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#538]

**Comment by gbinal** [comment]
I'm working with @adelevie on this right now.  

@adelevie - #541, #539, #537 are the last issues we'll need to factor in.  In summary, they: 
1) Further pare down which scans are run.  
2) The `Average No. of Errors` column will be hidden.  It's up to us as to whether to keep it or drop it from the data files themselves.  
3) The `Form Errors` and `Other Errors` columns will be hidden.  It's up to us as to whether to keep it or drop it from the data files themselves.  
4) For the remaining three columns (`Color Contrast`, `HTML/Attribute`, and `Alt Tag`), *we need to switch them from Totals to Averages*.

I'm looking into 1-3, but we'll need to factor in 4 for sure in the scripts.  Just to be clear, the denominator (when dividing by the number of domains), should be the number of live, non-redirecting domains for the agency - in other words, the number of domains one sees for an agency when [filtering on the Domains tab](https://staging.pulse.cio.gov/accessibility/domains/#q=agency%20name).

**Comment by gbinal** [comment]
Thinking through this in order, I need to:
1) update the scanning process to [remove these 2 scans](https://github.com/18F/pulse/issues/541)
  a) Make sure that the scans I'm adding to the ignore list are properly formatted.  
2) ensure that none of the remaining scans are in the `Forms` or `Other` error categories. 
3) successfully run a new scan 
4) Update the scripts to: 
  a) remove the `Form` and `Other` categories
  b) Rename the Rename `Alt Tag Errors` to `Missing Image Descriptions`
  c) For the `agencies.json` file, compute averages instead of totals for each column.

**Comment by gbinal** [comment]
Note that step 3 generates a file like this - 
[a11y_only_federal.csv.zip](https://github.com/18F/pulse/files/483588/a11y_only_federal.csv.zip)

**Comment by adelevie** [comment]
Wait ... How'd you get a zip file?

**Comment by gbinal** [comment]
I zipped it myself in order to upload it in the comment.  Interestingly, github wouldn't let me upload a csv.  

Sorry - I wasn't clear there.  I'll update the comment.

**Comment by gbinal** [comment]
Thanks to everyone's help on this.  There are still some outstanding issues I need to iron out but it's at least a repeatable process without needing devs.  I'll close this and open new issues for the remaining items.
